big_tests/src/ct_mongoose_log_hook.erl

```diff
@@ -110,7 +110,7 @@ terminate(State) ->
 spawn_log_reader(Node, Cookie) ->
     %% Set cookie permanently
     erlang:set_cookie(Node, Cookie),
-    AbsName = rpc:call(Node, filename, absname, ["log/ejabberd.log"], 5000),
+    AbsName = rpc:call(Node, filename, absname, ["log/mongooseim.log.1"], 5000),
```

Comment:
I'm curious where did this `.1` come from?

Reply:
The logs are written in rotating files, that is, when one is full, we jump to the next one, in a circle, renaming all files accordingly so that we always write to the same one. Lager was keeping the live file without the `.1` suffix, and all the others with the suffixes `.2`, `.3`, etc. Logger keeps all of them suffixed, including the live one. Don't really know if that's possible to change 🤔